src/utils/s3Utils.ts

```diff
@@ -0,0 +1,37 @@
+import { GetObjectCommand, PutObjectCommand } from '@aws-sdk/client-s3';
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+import { s3Client } from '../libs/s3Client';
+import { uniqueString } from './utils';
+
+/**
+ * Upload an object to s3
+ * @param data
+ * @returns
+ */
+export const uploadObject = async (data, fileName = '') => {
+  const name = fileName || uniqueString();
+  const bucketParams = {
+    Bucket: process.env.S3_BUCKET,
+    Key: name,
+    Body: data,
+  };
+
+  try {
+    const data = await s3Client.send(new PutObjectCommand(bucketParams));
+    return { key: name, data };
+  } catch (err) {
+    console.log('Error', err);
+  }
+};
+
+export const getS3ObjectUrl = async (filename) => {
+  const params = {
+    Bucket: process.env.S3_BUCKET,
+    Key: filename,
+  };
+  const command = new GetObjectCommand(params);
+
+  // 3600 = 1day
+  const url = await getSignedUrl(s3Client, command, { expiresIn: 3600 });
```

Comment:
I think we can use simple access URLs that are direct links to S3 objects, image files because we will save each link in DB. They should be static and permanently accessible.
   
btw, I noticed we should have places' master data for the Discover feature....

Reply:
@koh1project For security I thought it would be better just to store the image name inside DB and whenever the user access it, we will create the URL and send it to the front end. We also don't want users who do not have access to other user profiles to view their photos, even if they have the link. 
Still if you think it would be better to use public URLs let me know.

Reply:
Images will be not only one from user uploading, but also just static images for places, such as parks, restaurants. They are static but connected with DB information, so if we want to keep users' image secret, it would be good to have another bucket with different policy for those non-confidential images.

Reply:
@jatinparmar96 
But for this reason, I am ok with this implementation.
After merging it, please share your ENV file including S3 information.